config/locale/de.json

```diff
@@ -1,39 +1,204 @@
 {
+	"Oops, both Great and Ultra league parameters were set in command! - check the":"Fehler: Im Befehl wurden sowohl Super- als auch Hyperliga-Parameter angegeben. Siehe:",
+	"Oops, a distance was set in command but no location is defined for your tracking - check the":"Fehler: Im Befehl wurde eine Distanz angegeben, aber es wurde noch kein Standort festgelegt. Siehe:",
+	"Oops, no distance was set in command and no area is defined for your tracking - check the":"Fehler: Im Befehl wurde keine Distanz angegeben, aber es wurde noch kein Gebiet festgelegt. Siehe:",
+	"Please run commands in Direct Messages":"Befehle müssen als Direktnachrichten an den Poracle-Bot gesendet werden!",
+	"Current language is set to":"Aktuell eingestellte Sprache:",
+	"I only recognise the following languages":"Poracle unterstützt nur folgende Sprachen:",
+	"I have changed your language setting to":"Folgende Sprache wurde eingestellt:",
+	"You have reached the limit of {0} messages over {1} seconds":"Das Limit von {0} Benachrichtigungen innerhalb von {1} Sekunden wurde erreicht.",
+	"english":"Englisch",
+	"french":"Französisch",
+	"spanish":"Spanisch",
+	"italian":"Italienisch",
+	"german":"Deutsch",
+	"japanese":"Japanisch",
+	"korean":"Koreanisch",
+	"polish":"Polnisch",
+	"russian":"Russisch",
+	"name":"Name",
+	"does not seem to be registered. add it with":"scheint nicht registriert zu sein. Hinzufügen mittels",
+	"webhook":"webhook",
+	"add":"hinzufügen",
+	"<Your-Webhook-url>":"<Webhook-URL>",
+	"channel":"kanal",
+	"no valid areas there, please use one of":"Fehler: Kein gültiges Gebiet angegeben. Folgende Gebiete sind verfügbar:",
+	"Added areas:":"Hinzgefügte Gebiete:",
+	"Removed areas:":"Entfernte Gebiete",
+	"Current configured areas are:":"Aktuell sind folgende Gebiete festgelegt:",
+	"nameYourWebhookName":"webhookName",
```

Comment:
the name in front here is the command parameter so needs to be there
eg namewebhookName

Reply:
mhh not sure wheter I really understand this, but done anyway.

btw: that's one of the occasions where something isn't translated:

```
return await msg.reply(`Webhook ${target.name} ${client.translator.translate('does not seem to be registered. add it with')} ${client.config.discord.prefix}${client.translator.translate('webhook')} ${client.translator.translate('add')} ${client.translator.translate('<Your-Webhook-url>')} ${client.translator.translate('nameYourWebhookName')} nameYourWebhookName`)
```